flink-runtime/src/test/java/org/apache/flink/runtime/io/network/partition/ResultPartitionFactoryTest.java

```diff
@@ -105,6 +105,7 @@ private static ResultPartition createResultPartition(
 		final ResultPartitionDeploymentDescriptor descriptor = new ResultPartitionDeploymentDescriptor(
 			new PartitionDescriptor(
 				new IntermediateDataSetID(),
+				1,
```

Comment:
Looks like, I overlooked this non-builder usage, updated https://github.com/azagrebin/flink/commit/1c28b2bcb82ee8aa8e3cdf53e6cbc890715205c7